src/modules/workspaces/workspaces.controller.ts

```diff
@@ -489,6 +490,55 @@ export class WorkspacesController {
     );
   }
 
+  @Get('/:workspaceId/my-trash')
```

Comment:
Why 'my-trash' and just not 'trash'. In fact, listing the trash should be like listing files and folders as usual but using a query param with the status ('TRASHED') so we do not have to keep multiple endpoints for the same thing